setup.py

```diff
@@ -267,7 +267,6 @@ def get_torch_arch_list() -> Set[str]:
             "cxx": CXX_FLAGS,
             "nvcc": NVCC_FLAGS,
         },
-        libraries=["cuda"],
```

Comment:
You shouldn't remove this. I used cuda driver's API and must link cuda driver for it to work.